build.cake

```diff
@@ -281,86 +183,63 @@ Task("CheckForError")
 
 Task("Test45")
     .Description("Tests the .NET 4.5 version of the framework")
-    .IsDependentOn("Build45")
+    .IsDependentOn("Build")
     .OnError(exception => { ErrorDetail.Add(exception.Message); })
     .Does(() =>
     {
-        var runtime = "net-4.5";
+        var runtime = "net45";
         var dir = BIN_DIR + runtime + "/";
         RunNUnitTests(dir, FRAMEWORK_TESTS, runtime, ref ErrorDetail);
```

Comment:
Given all this is rewritten and working on different platforms, could we check if the InProcess workaround flag is still needed for Travis, which is inside this RunNUnitTests method? Again - could wait till clean up if we'd prefer.

Reply:
Thanks, added to my list. I'd prefer to wait and do it with a later cleanup PR.